src/service/programService.js

```diff
@@ -31,7 +32,7 @@ var async = require('async')
 
 const queryRes_Max = 1000;
 const queryRes_Min = 300;
-const stackTrace_MaxLimit = 500;
+//const stackTrace_MaxLimit = 500;
```

Comment:
Why did you removed this? We should not have very big log message..

Reply:
May be you can update the value if required for errors.

Reply:
@vinukumar-vs This variable was not being used in this file. That is why removed.